sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -142,7 +142,7 @@ prediction or predictions about a single text input. An operation's result type
 about the input document and how it was processed. An operation result contains a `isError` property that allows to
 identify if an operation executed was successful or unsuccessful for the given text input. When the operation results
 an error, you can simply call `getError()` to get `TextAnalyticsError` which contains the reason why it is unsuccessful. 
-If you are interested in how many characters in your input text or number of operation transactions been going through,
+If you are interested in how many characters in your document or number of operation transactions been going through,
```

Comment:
how many characters are in your document, or the number of operation transactions that have gone through